lib/bs58/bs58.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
```

Comment:
Where does this file originate? Can you add license clauses in doccomments at the top of each file?

Reply:
This is from the bs58check package. I'll add the license clausules to the files that don't have it already.

Reply:
Actually, can I just add a reference to the license in the files? Adding the whole license clausules will increase the size of the package.

Reply:
@inspireme6 Yes, that's fine - something along the lines of:
```
// bs58check - A straight forward implementation of base58check extending upon bs58.
// Extracted from the [library name] found at [library URL].
// This code is licensed under the [license name], which can be found at [license URL].
```